src/app/components/text-image-component/text-image-component.component.ts

```diff
@@ -0,0 +1,19 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-text-image-component',
+  templateUrl: './text-image-component.component.html',
+  styleUrls: ['./text-image-component.component.css']
+})
+export class TextImageComponentComponent implements OnInit {
+
+  constructor() { }
+
+  @Input() text: string;
+  @Input() image: string;
+  @Input() position: boolean;
```

Comment:
I forgot in the YouTrack task about the title for each component.
Add a 4th Input called title (a string) and display it above the text. Again use https://dscupt.tech/ for reference